akamaiwaf.json

```diff
@@ -1561,42 +1562,92 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "get siteshields",
+            "description": "Get akamai site shields ip ranges",
+            "type": "investigate",
+            "identifier": "get_siteshields",
+            "read_only": true,
+            "parameters": {},
```

Comment:
suggestion: Seeing that we don't need to provide `siteshields` in the apram, should this action be called `list siteshields` instead?

Seem like this would be consistent with other apps as well. See [reference](https://github.com/search?q=org%3Asplunk-soar-connectors+%22list%22&type=code)

Reply:
Hello @siisar, Could you please address the suggested change? Thank you!

Reply:
Hi, absolutely makes sense, feel free to make that change...

Reply:
Modified the action name to 'list siteshields'.